validphys2/src/validphys/tests/test_pythonmakereplica.py

```diff
@@ -49,7 +49,6 @@ def test_commondata_unchanged(data_config, dataset_inputs, use_cuts):
     pre_mkrep_cvs = [deepcopy(cd.central_values) for cd in ld_cds]
     pre_mkrep_sys_tabs = [deepcopy(cd.systematics_table) for cd in ld_cds]
     pre_mkrep_cd_tabs = [deepcopy(cd.commondata_table) for cd in ld_cds]
-    API.make_replica(**config)
```

Comment:
why?

Reply:
Ah yes, I wasn't paying attention to context